virtool/uploads/models.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import Column, String, Boolean, Integer, DateTime
+
+from virtool.postgres import Base
+
+
+class Upload(Base):
+    __tablename__ = "uploads"
+
+    id = Column(Integer, primary_key=True)
+    created_at = Column(DateTime)
+    field = Column(String)
+    name = Column(String, unique=True)
```

Comment:
Will not be unique. The user may upload the same file twice. This is why we use `name_on_disk` to keep the upload unique when the file name may be the same.